router/router.go

```diff
@@ -13,6 +13,7 @@ func NewRouter(db database.Database) http.Handler {
 		userRoute.Post("/signup", SignUp(db))
 		userRoute.Post("/login", Login(db))
 		userRoute.Put("/passcode", UpdatePasscode(db))
+		userRoute.Get("/all", FetchAllUsers(db))
```

Comment:
This isn't RESTful
```suggestion
		userRoute.Get("/", FetchAllUsers(db))
```

Reply:
ohh  okay. it was a bit confusing cos the main path is `/user` and the endpoints fetches all users. I'll make the change thanks